daprdocs/content/en/developing-applications/develop-components/pluggable-components/develop-pluggable.md

```diff
@@ -105,6 +105,167 @@ After generating the above state store example's service scaffolding code using
 
 This concrete implementation and auxiliary code are the **core** of your pluggable component. They define how your component behaves when handling gRPC requests from Dapr.
 
+## Return semantic state store errors
+
+In order to meet concurrency requirements, your state store pluggable components need error mapping, from gRPC returned errors to semantic errors. Each of the three Dapr state store errors below maps to a gRPC status code. 
```

Comment:
This description is targeting EtagMismatch error only, as we have others I think we can use this as a `Description` column, how about?

Reply:
I would say that we should use this initial paragraph to describe the goal of returning semantic errors. e.g

```suggestion
Returning semantic errors are also part of the pluggable component protocol. The component must return specific gRPC codes that have semantic meaning for the user application, those errors are used to a variety of situations from concurrency requirements to informational only.
```